src/xrCore/LocatorAPI.cpp

```diff
@@ -1931,29 +1935,6 @@ bool CLocatorAPI::can_modify_file(pcstr fname)
     }
 }
 
-bool CLocatorAPI::folder_exist(pcstr fullpath) // only Win ?
```

Comment:
The previous commit adds the function, the current commit removes it... Merge the changes, don't clutter up the history!